data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -419,7 +419,7 @@ fire:  +10%"
         [/message]
         [message]
             speaker="Li'sar"
-            message= _ "My first use for it is going to be to help us get out of this hole! I hope you consider that wise."
+            message= _ "My first use for it is going to be to help us get out of this hole! I hope you consider that wise, Cousin."
```

Comment:
I'd say drop the "Cousin" on this one. I believe at this point in time, Li'sar is still less friendly with Konrad and co., so she'll refer to them somewhat brusquely.

Reply:
Actually, the first thing Li'sar thing at the start of the next scenario is that [she'd let Konrad live](https://github.com/wesnoth/wesnoth/blob/ec3b2fb381d17a9297ae5f9a2c1ff9fa16649b75/data/campaigns/Heir_To_The_Throne/scenarios/18_A_Choice_Must_Be_Made.cfg#L210-L217). She's becoming _more_ friendly than she was [at the start of S16](https://github.com/wesnoth/wesnoth/blob/ec3b2fb381d17a9297ae5f9a2c1ff9fa16649b75/data/campaigns/Heir_To_The_Throne/scenarios/16_Hasty_Alliance.cfg#L214-L237), not less.

Reply:
Not saying she's becoming less friendly, it's that she isn't very friendly with them at this point. Say on a scale of 1-10, her friendliness is increasing, but it's still at like a 3.

Reply:
When I read the three suggestions, I didn't realise this was a utils file and so I'm guessing it's actually used a lot earlier than the other two instances in S18 and S21. I agree that if it's that much earlier in the campaign then they would not yet have reached familiarity/friendliness.

However, the revision to the second instance suggested above sounds okay. While I like that Li'sar and Konrad become friendly over the course of the campaign, and these changes are nice, maybe we don't want to spend too much time stressing the cousin relationship because by the end we find out they are not related after all (making their subsequent marriage less awkward - real-world weddings between royal cousins notwithstanding).

I just had a thought: `Delfador! Li’sar is my cousin - and more importantly, our ally. I don’t want to fight her!` Noting that later on Konrad denies to the elves hinting that there may be more to their relationship than just being friends.

Reply:
> When I read the three suggestions, I didn't realise this was a utils file and so I'm guessing it's actually used a lot earlier than the other two instances in S18 and S21. I agree that if it's that much earlier in the campaign then they would not yet have reached familiarity/friendliness.

Your guess is wrong. That text is used in the victory event of S17 if Li'sar takes the Sceptre of Fire; it's right before the start event of S18 where one of the other changes is.

> However, the revision to the second instance suggested above sounds okay. While I like that Li'sar and Konrad become friendly over the course of the campaign, and these changes are nice, maybe we don't want to spend too much time stressing the cousin relationship because by the end we find out they are not related after all (making their subsequent marriage less awkward - real-world weddings between royal cousins notwithstanding).

:thinking: As you said, Konrad already mentions to the elves that Li'sar is his cousin. Does saying that to Delfador too qualify as "spending too much time stressing the cousin relationship"?